components/script/dom/trustedtypepolicyfactory.rs

```diff
@@ -369,7 +369,7 @@ impl TrustedTypePolicyFactoryMethods<crate::DomTypeHolder> for TrustedTypePolicy
     /// <https://www.w3.org/TR/trusted-types/#dom-trustedtypepolicyfactory-isscripturl>
     #[allow(unsafe_code)]
     fn IsScriptURL(&self, cx: JSContext, value: HandleValue) -> bool {
-        unsafe { root_from_handlevalue::<TrustedScriptURL>(value, *cx).is_ok() }
+        unsafe { root_from_handlevalue::<TrustedScriptURL>(value, cx).is_ok() }
```

Comment:
Can you remove the `unsafe` block here and in other places that you call `root_from_handlevalue` now that it is not an unsafe function? Please give that a shot.